graphql/resolvers.go

```diff
@@ -1346,53 +1369,29 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 				formattedURL := fmt.Sprintf("%s%s", r.sc.GetURL(), *apiTest.Logs.RawDisplayURL)
 				apiTest.Logs.RawDisplayURL = &formattedURL
 			}
-			baseTestStatus := baseTestStatusMap[*apiTest.TestFile]
+			baseTestStatus := baseTestStatusMap[utility.FromStringPtr(apiTest.DisplayTestName)]
```

Comment:
in pre #4564 test results the display name might be empty.
Maybe we should check _both_ the `TestFile` and `DisplayTestName`? (I thought this was the plan)

Reply:
Good catch, checking TestFile is necessary to maintain backwards compatibility with older cedar results. I think it would be ideal if we didn't have to support the legacy TestFile key in the app code since these days TestFile is a random string. I made a ticket to change this here: https://jira.mongodb.org/browse/EVG-15136